dashboard/config/routes.rb

```diff
@@ -27,12 +27,10 @@
 
   get "/congrats", to: "congrats#index"
 
-  resources :gallery_activities, path: '/gallery' do
-    collection do
-      get 'art', to: 'gallery_activities#index', app: Game::ARTIST
-      get 'apps', to: 'gallery_activities#index', app: Game::PLAYLAB
-    end
-  end
+  # Redirects to /projects
+  # TODO: (ErinB) remove when gallery_activities are fully deleted
+  get "/gallery", to: "gallery_activities#index"
```

Comment:
we can redirect directly from the router ([rails source](https://guides.rubyonrails.org/routing.html#redirection)) like this:

```
get '/gallery', to: redirect('/projects')
```

then we can totally remove the `gallery_activities#index` method